site-v15/assets/atlas-runtime.css

```
@@ -0,0 +1,11 @@
+.atlas-filters{display:flex;flex-wrap:wrap;gap:.55rem;margin:0 0 1rem;padding:.8rem;border:1px solid rgba(170,214,197,.22);border-radius:.85rem;background:rgba(5,14,18,.64)}
+.atlas-filters legend{width:100%;padding:0 .2rem .55rem;font-size:.72rem;font-weight:800;letter-spacing:.08em;text-transform:uppercase}
+.atlas-filter-option{display:inline-flex;align-items:center;gap:.42rem;min-height:2.35rem;padding:.48rem .72rem;border:1px solid rgba(170,214,197,.22);border-radius:999px;cursor:pointer}
+.atlas-filter-option:has(input:checked){border-color:currentColor;background:rgba(170,214,197,.1)}
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)